src/System.Net.WebSockets.Client/tests/System.Net.WebSockets.Client.Tests.csproj

```diff
@@ -15,7 +15,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
-      <Link>Common\System\PlatformDetection.cs</Link>
+      <Link>Common\tests\System\PlatformDetection.cs</Link>
```

Comment:
Please settle on one of the methods (below is still in the CommonTestPath yet it isn't in the \tests\ folder). Since this is already a test project, the Common folder in Visual Studio should be fine.